src/main/java/com/epam/facade/Facade.java

```diff
@@ -56,6 +59,17 @@ public List<Training> getAllTrainings() {
             log.error("Error fetching all trainings.", e);
             throw e;
         }
+    }
+
+    public void saveTraineeFromTraineeJsonData(TraineeJsonDto traineeJsonDto) {
+        Trainee trainee = Mappers.convertTraineeJsonDtoToTrainee(traineeJsonDto);
```

Comment:
You could have methods for mapping right in the traineeJsonDto like
```java
public static Trainee toTrainee(TraineeJsonDto dto);
public static TraineeJsonDto ofTrainee(Trainee trainee);
```
But this is pretty opinionated. Leaving this here just FYI.